### ISSUE TITLE
UPDATE: README.md file

### ISSUE THREAD

**Issue by uisendy** [skyv26/portfolio#16]
Hello [skyv26](https://github.com/skyv26). Very Awesome works in this portfolio project. It is a high flying project. 🚀 

However i noticed some issues on your README.md file that i have raised and needs reevaluation as below.

### SUGGESTIONS ♻️ 
- I strongly suggest you update your description, lets know the objective and goal of the project as well as the scope and what i can learn from it. with this, anyone who visits the page would know before hand what he/she would learn and expect to see. 
- Secondly, you should add a contributor to the page so that anyone who finds a mistake as small as a typo can feel rightful to raise and issue on the page and you can make correction. 

### The last but not the least is an OPTIONAL review. 
- I suggest if you do have a someone who took part in the project, you can acknowledge the person or account, 
- Also, you can make your contact section more robust by adding your social media links or other webpages where a visitor can get to know more about you or contact with ease. 

In conclusion, you main work is a awesome work and i commend it a lot. The few suggestions can make the work even more and more better.  

## Cheers 🥂 🎈  and Happy Coding 💻

**Comment by skyv26** [comment]
I have successfully updated my README.md file with below updation:

- Added the acknowledgment with name of my coding partner of the project because of course he contributed his work in my portfolio.
- Added the description of portfolio in order to define the my interest behind making this portfolio.
- I also added my social platform links so that anyone can find me and connect with me easily.
